torchmdnet/datasets/ani.py

```diff
@@ -134,18 +149,16 @@ def sample_iter(self, mol_ids=False):
                     if data := self.filter_and_pre_transform(data):
                         yield data
 
-    def get_atomref(self, max_z=100):
-        """Atomic energy reference values for the :py:mod:`torchmdnet.priors.Atomref` prior."""
-        refs = pt.zeros(max_z)
-        refs[1] = -0.500607632585 * self.HARTREE_TO_EV  # H
-        refs[6] = -37.8302333826 * self.HARTREE_TO_EV  # C
-        refs[7] = -54.5680045287 * self.HARTREE_TO_EV  # N
-        refs[8] = -75.0362229210 * self.HARTREE_TO_EV  # O
-
-        return refs.view(-1, 1)
-
 
 class ANI1XBase(ANIBase):
+    __doc__ = ANIBase.__doc__
+    _ELEMENT_ENERGIES = {
```

Comment:
I think moving this to the base class might be wrong actually. ANI1CCX also is a child of this class but I'm relatively sure the reference energies of coupled cluster cannot match the ones of the DFT of ANI-1x